test/e2e/util.go

```diff
@@ -70,9 +70,9 @@ func waitForPodCondition(c *client.Client, ns, podName, desc string, condition p
 		if done {
 			return err
 		}
-		Logf("Waiting for pod %s status to be %q (found %q) (%d secs)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
+		Logf("Waiting for pod %s status to be %q (found %q) (%.2f seconds)", podName, api.PodRunning, pod.Status.Phase, time.Since(start).Seconds())
 	}
-	return fmt.Errorf("gave up waiting for pod %s to be %s after %v", podName, desc, podStartTimeout)
+	return fmt.Errorf("gave up waiting for pod %s to be %s after %.2f seconds", podName, desc, podStartTimeout.Seconds())
```

Comment:
`%.2f` is fine. FWIW, `time.Duration` has an intelligent value printer, so printing the duration directly with `%v` also results in reasonable output of the form `30s, 5m0s`, etc, though the output was temporarily borked prior to #5134. Still, having it consistently print the timeout in seconds is worthwhile, so merged.